src/commands/publish.ts

```diff
@@ -35,13 +38,22 @@ const publish = async (
 
     createBlockRequest({ displayName, publishedName }, blockData, category)
         .then((res: AxiosResponse) => {
+            const sortedVersions = sortVersions(res.data.versions);
+            const currentVersion = sortedVersions[0].version;
+
             updateBlockSettingsFile({
                 category,
+                currentVersion,
                 displayName,
                 id: res.data.id,
+                isActive: true,
                 isPublic: false,
+                isStaging: true,
```

Comment:
What's the motivation to keep track of all this in the file? isActive, isStaging. Isn't the version enough?

Reply:
We do need to keep track of versions though for checking the type of actions they can use. I was pulling out the needed flags for simplicity but I can just save the version in the file and go from there.